src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => (
```

Comment:
Add the 'import React' statement at the beginning of the file.

Reply:
if I import react lint throws an error